ansible/vars/default.yml

```
@@ -1,12 +1,10 @@
 ---
 
-tljh_plasma: git+https://github.com/plasmabio/plasma@master#"egg=tljh-plasma&subdirectory=tljh-plasma"
-tljh_repo2docker: tljh-repo2docker==1.0.1
+tljh_plasma: git+https://github.com/jtpio/plasma@updates#"egg=tljh-plasma&subdirectory=tljh-plasma"
```

Update before merging:

```suggestion
tljh_plasma: git+https://github.com/plasmabio/plasma@master#"egg=tljh-plasma&subdirectory=tljh-plasma"
```